ui/raidboss/data/06-ew/raid/p9n.ts

```diff
@@ -8,7 +11,166 @@ const triggerSet: TriggerSet<Data> = {
   id: 'AnabaseiosTheNinthCircle',
   zoneId: ZoneId.AnabaseiosTheNinthCircle,
   timelineFile: 'p9n.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'P9N Gluttony\'s Augur',
+      type: 'StartsUsing',
+      netRegex: { id: '8116', source: 'Kokytos', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P9N Global Spell',
+      type: 'StartsUsing',
+      netRegex: { id: '8141', source: 'Kokytos', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'aoe + bleed',
+          de: 'AoE + Blutung',
+          fr: 'AoE + saignement',
+          ja: 'AoE + DoT',
+          cn: 'AOE + 流血',
+          ko: '전체 공격 + 도트',
+        },
+      },
```

Comment:
@quisquous There's been a lot of "AoE + bleed" this expansion, I think we should promote this up to Responses.

Reply:
Happy to make a follow-up PR to do that after this is merged.